test/it/policies.js

```diff
@@ -0,0 +1,253 @@
+      status: 'ACTIVE',
+      name: 'node-sdk: GetPolicy',
+      description: 'The default policy applies in all situations if no other policy applies.',
+    };
+    const oktaSignOnPolicy = new models.OktaSignOnPolicy(policy, client);
+    const createdPolicy = await client.createPolicy(oktaSignOnPolicy);
+    const retrievedPolicy = await client.getPolicy(createdPolicy.id);
+    await client.deletePolicy(createdPolicy.id);
+
+    expect(retrievedPolicy).to.not.be.undefined;
+    expect(retrievedPolicy.name).to.equal(policy.name);
+    expect(retrievedPolicy.type).to.equal(policy.type);
+    expect(retrievedPolicy.status).to.equal(policy.status);
+    expect(retrievedPolicy.description).to.equal(policy.description);
+  });
+
+  it('create sign on policy with group condition', async () => {
+      // 1. Create a new group
+    const newGroup = {
+      profile: {
+        name: 'Get Test Group'
+      }
+    };
+
+    // Cleanup the group if it exists
+    await utils.cleanup(client, null, newGroup);
+
+    const createdGroup = await client.createGroup(newGroup);
+    utils.validateGroup(createdGroup, newGroup);
+
+    // 2. Set Up Policy JSON
+    const policy = {
+      type: 'OKTA_SIGN_ON',
+      status: 'ACTIVE',
+      name: 'node-sdk: CreateOktaSignOnPolicyWithConditions',
+      description: 'The default policy applies in all situations if no other policy applies.',
+      conditions: {
+        people: {
+          groups: {
+            include: [
+              createdGroup.id
+            ]
+          }
+        }
+      }
+    };
+    const oktaSignOnPolicy = new models.OktaSignOnPolicy(policy, client);
+    const createdPolicy = await client.createPolicy(oktaSignOnPolicy);
+    await client.deletePolicy(createdPolicy.id);
+
+    expect(createdPolicy).to.not.be.undefined;
+    expect(createdPolicy.name).to.equal(policy.name);
+    expect(createdPolicy.type).to.equal(policy.type);
+    expect(createdPolicy.status).to.equal(policy.status);
+    expect(createdPolicy.description).to.equal(policy.description);
+    expect(createdPolicy.conditions.people.groups.include).to.not.be.empty;
+    expect(createdPolicy.conditions.people.groups.include[0]).to.equal(createdGroup.id);
+  });
+
+  it('can get policy by type', async () => {
+    const policy1 = {
+      type: 'OKTA_SIGN_ON',
+      status: 'ACTIVE',
+      name: 'node-sdk: Sign On GetAllPoliciesByType',
+      description: 'The default policy applies in all situations if no other policy applies.',
+    };
+
+    const policy2 = {
+      type: 'PASSWORD',
+      status: 'ACTIVE',
+      name: 'node-sdk: Password GetAllPoliciesByType',
+      description: 'The default policy applies in all situations if no other policy applies.',
+    };
+
+    const oktaSignOnPolicy1 = new models.OktaSignOnPolicy(policy1, client);
+    const createdPolicy1 = await client.createPolicy(oktaSignOnPolicy1);
+
+    const oktaSignOnPolicy2 = new models.OktaSignOnPolicy(policy2, client);
```

Comment:
since `policy2` is a password policy, is it correct to create a new `SignOnPolicy` here? I don't remember how it works.